specification/servicebus/ServiceBus.Management/SBAuthorizationRule.tsp

```diff
@@ -0,0 +1,443 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+import "./SBNamespace.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.ServiceBus;
+/**
+ * Description of a namespace authorization rule.
+ */
+#suppress "@azure-tools/typespec-azure-core/casing-style" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@parentResource(SBNamespace)
+model SBAuthorizationRule
+  is Azure.ResourceManager.ProxyResource<SBAuthorizationRuleProperties> {
+  ...ResourceNameParameter<
+    Resource = SBAuthorizationRule,
+    KeyName = "authorizationRuleName",
+    SegmentName = "authorizationRules",
+    NamePattern = ""
+  >;
+
+  /**
+   * The geo-location where the resource lives
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @visibility(Lifecycle.Read)
+  location?: string;
+}
```

Comment:
Please confirm with service team if `location` is required or optional? if it is required,  use `TrackedResource` instead of `ProxyResource`?

Reply:
Confirmed it's optional